apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -642,7 +642,7 @@ func (m *ModelManagerAPI) ListModelSummaries(req params.ModelSummariesRequest) (
 		return result, errors.Trace(err)
 	}
 
-	modelInfos, err := m.state.ModelSummariesForUser(userTag, req.All)
+	modelInfos, err := m.state.ModelSummariesForUser(userTag, req.All && m.isAdmin)
```

Comment:
same here?

Reply:
this is just replicating the original logic